app/presenters/admin/export/document_presenter.rb

```diff
@@ -0,0 +1,44 @@
+class Admin::Export::DocumentPresenter < Whitehall::Decorators::Decorator
```

Comment:
Hmm, if I move the presenter out of the Admin namespace, but leave the controller in the `Admin` namespace, then I'll have to require the DocumentPresenter in the controller.  Is this what you mean?

Reply:
No sorry that's not what I meant. 

I meant that many of the presenters that lack an admin namespace are already used in an admin context so giving them this namespace would be a potential point of confusion and shouldn't add any value since the Export namespace should suffice.

Although looking at the code the admin controllers only seem to use one presenter:

```
➜  admin git:(add-document-export-api) ✗ ag Presenter
roles_controller.rb
68:    RoleTypePresenter.role_attributes_from(params[:role][:role_type])[:type]
```

you shouldn't have to do anything with requiring or scoping class names

Reply:
I see, thanks for the explanation 👍.  Made the change and pushed the update.